interface/super/edit_globals.php

```diff
@@ -367,7 +374,7 @@ function validate_file(){
 <?php
 $i = 0;
 foreach ($GLOBALS_METADATA as $grpname => $grparr) {
-  if ( $_GET['mode'] != "user" || ($_GET['mode'] == "user" && in_array($grpname, $USER_SPECIFIC_TABS)) ) {
+  if ( !$userMode || ($userMode && in_array($grpname, $USER_SPECIFIC_TABS)) ) {
```

Comment:
This is equivalent:
if (!$userMode || in_array($grpname, $USER_SPECIFIC_TABS)) {